### PR TITLE
fix: app crash on clicking submit loanbutton in apply for loan fragment

### LINKED ISSUE #220: Bug: App crashes when submit button is clicked in ApplyForLoan activity with no internet connection
**Summary:**
Bug: App crashes when submit button is clicked in ApplyForLoan activity with no internet connection

**Steps to reproduce:**
1. Log in the app
2. Goto Loan fragment
3. Turn off the internet connection.
4. goto "apply for loan" in the action bar(menu)
5. Click the submit button
6. the app crashes.

**Expected behavior:**
app should not crash on clicking submit button. Rather it should display some snackbar when no internet connection

**Observed behavior:**
app crashes on clicking submit button

**Device and Android version:**
Android 5.1.1  xiaomi redmi prime2

**Screenshots:**
.

### PR DESCRIPTION
**Pull request by itsamitgoel**
Fixes #220  say user is in home screen and then losses his internet connection. then going to Apply for Loan fragment and clicking SUBMIT LOAN button causes the app to crash. fixed it .Instead it now shows a snackbar with "failed to load template" message.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by itsamitgoel** [comment]
screenshot_2017-03-31-02-29-07-854_org mifos selfserviceapp

**Comment by itsamitgoel** [comment]
Fixes #220

File: app/src/main/java/org/mifos/selfserviceapp/ui/fragments/LoanApplicationFragment.java [hunk 1]
```diff
@@ -119,31 +120,38 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+            loansPayload.setTransactionProcessingStrategyId(
+                    loanTemplate.getTransactionProcessingStrategyId());
+            loansPayload.setInterestCalculationPeriodType(
+                    loanTemplate.getInterestCalculationPeriodType().getId());
+            loansPayload.setInterestType(loanTemplate.getInterestType().getId());
+
+            loanApplicationPresenter.createLoansAccount(loansPayload);
+
+        } else {
+            showError("failed to load template");
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/selfserviceapp/ui/fragments/LoanApplicationFragment.java:153]: This feature needs more improvement but for now change SnackBar message to strings.xml string's `no_internet_connection`

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by dilpreet2028** [comment]
Issue fixed in #300

**Pull request closed without merging**